search/tests/test_indexing.py

```diff
@@ -205,11 +205,11 @@ def get_count():
             return count
 
         set_cache_timeout(0)
```

Comment:
Since both course and vocabularies are queried in `get_count()` (above), there is still value in explicitly enabling and disabling caching for the `self.assertNumQueries` tests (below).